portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const SPOTIFY_LINK = "https://open.spotify.com/playlist/6rmiBWfdA4jGdTWaYMAQri";
+
+
+/**
+ * Sends user a spotify link
+ */
+function sendSpotifyLink(){
+    const linkContainer = document.getElementById("spotify-link-container");
+    linkContainer.innerHTML = "<em>Jazz isn't an avenue I've explored!</em> Here is the spotify link: \n" + SPOTIFY_LINK;
+}
+
+
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomMovie() {
+  const movies =
```

Comment:
Move this to a top-level const like you did for SPOTIFY_LINK (good job on the casing there!).

You can also add descriptions, if you think it needs one, and a type annotation.
ex:
```
/**
 * The description of my top level constant.
 * @type {string}
 */
const VAR_NAME = 'var_value';
```